2020/hamming-codes/spanish/sentence_translations.json

```
@@ -20,7 +20,7 @@
  {
   "input": "There is a whole pile of mathematical cleverness that allows us to store data, and just as importantly to transmit data, in a way that's resilient to errors.",
   "model": "nmt",
-  "translatedText": "Existe una gran cantidad de inteligencia matemática que nos permite almacenar datos y, lo que es igualmente importante, transmitirlos de una manera resistente a los errores.",
+  "translatedText": "Existe una gran cantidad de inteligencia matemática que nos permite almacenar datos y, lo que es igualmente importante, transmitirlos de una manera resiliente los errores.",
```

```suggestion
  "translatedText": "Existe una gran cantidad de inteligencia matemática que nos permite almacenar datos y, lo que es igualmente importante, transmitirlos de una manera resiliente a los errores.",
```
I believe you forgot the `a` when changing it to `resiliente`

---

Yep